graylog2-server/src/main/java/org/graylog2/rest/resources/alarmcallbacks/AlarmCallbackHistoryResource.java

```diff
@@ -0,0 +1,93 @@
+/**
+ * This file is part of Graylog.
+ *
+ * Graylog is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * Graylog is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with Graylog.  If not, see <http://www.gnu.org/licenses/>.
+ */
+package org.graylog2.rest.resources.alarmcallbacks;
+
+import com.codahale.metrics.annotation.Timed;
+import com.google.common.collect.Lists;
+import com.wordnik.swagger.annotations.Api;
+import com.wordnik.swagger.annotations.ApiOperation;
+import com.wordnik.swagger.annotations.ApiParam;
+import org.apache.shiro.authz.annotation.RequiresAuthentication;
+import org.graylog2.alarmcallbacks.AlarmCallbackHistory;
+import org.graylog2.alarmcallbacks.AlarmCallbackHistoryService;
+import org.graylog2.alerts.Alert;
+import org.graylog2.alerts.AlertService;
+import org.graylog2.database.NotFoundException;
+import org.graylog2.plugin.streams.Stream;
+import org.graylog2.rest.models.alarmcallbacks.AlarmCallbackHistoryListSummary;
+import org.graylog2.rest.models.alarmcallbacks.AlarmCallbackHistorySummary;
+import org.graylog2.shared.rest.resources.RestResource;
+import org.graylog2.shared.security.RestPermissions;
+import org.graylog2.streams.StreamService;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import javax.inject.Inject;
+import javax.ws.rs.GET;
+import javax.ws.rs.Path;
+import javax.ws.rs.PathParam;
+import javax.ws.rs.Produces;
+import javax.ws.rs.core.MediaType;
+import java.util.List;
+
+@RequiresAuthentication
+@Api(value = "AlarmCallbacks", description = "Manage stream alarm callback histories")
```

Comment:
Shouldn't it be called `AlarmCallbackHistory`? Sharing the name with `AlarmCallbacks` is confusing, specially on swagger.